demisto_sdk/tests/test_files/swagger_python.txt

```diff
@@ -564,7 +564,7 @@ def update_user_command(client, args):
 
 def test_module(client):
     # Test functions here
-    demisto.results('ok')
+    return_results('ok')
```

Comment:
Not exactly related. But I am not clear why this file is `.txt` extension.

Reply:
This is an old file. From `openapi-codegen` . I believe because the developer didn't want the lint to run on py file. I disabled the linting on test file. I will change it the file extension to py.